api-docs/sustainability-api.md

```diff
@@ -13,7 +13,49 @@ The Trackunit Sustainability API is a REST API that enables customers to access
+3. Daily: This is a non-aggregated limited set of most recent values (ex. 60 days). It provides one value per day for each of the recent days.
+
+## Concepts
+
+### Equipment and Energy Consumption
+The energy consumption of construction equipment is separated in 2 broad categories for the purpose of emissions calculations:
+
+* Fuel Consumption (l)
+* Electric Consumption (kWh)
+
+A third category is introduced to include (3) Plug-in Hybrid Electric equipment. This category will have both fuel consumption and electric consumption being tracked simultaneously, and the associated emissions will be totaled for both.
+Fuel consumption by itself is a broad category that includes all kinds of fuel, for example:
+
+* Diesel
+* Biodiesel
+* Gasoline
+* Propane (LPG)
+* Hydrogen
+
+#### All the most commonly used fuel types can and will be measured in litres (l) as default.
+
+Electric-powered machines consumption also exist with more than one type, and their consumption is always tracked in kWh consumed:
+
+* Battery-electric (rechargeable with plug-in)
+* Direct Plug-in (no batteries)
+
+We have built our data structure and API to allow tracking of all types of consumption described above.
+
+## Use-Cases
+
+1. The number 1 use-case is one within the contractor segment. Contractors need to estimate total and broken down emissions for each of a group of machines at a location (branch, yard, site, etc.), within a category, or for comparisons between categories. The Daily endpoint provides the broken down values per machine per day for the set of recent days. This broken down data can also be used upstream by OEMs and Rental Houses to provide contractor-specific insights to their end-customers. Including emissions on invoices is an example of how contractors can consume this information.
+
+
+2. Many companies have set targets for emissions. The Monthly endpoint provides the dashboard needed to track baselines, progress, impact of emissions reduction initiatives, etc. It provides a long term view of where we come from and can be used to make predictions for the future. This type of information is useful for periodic reporting to the board, for internal awareness, for reporting to investors, and to regulators if needed.
+
+
+3. Understanding trends of changing use and emissions rate as machines age is an important insight within a fast-changing commercial and regulatory environment. The Lifetime endpoint can provide a big picture view at the asset level to course adjustments in purchasing and dispatching to meet the marketplace where the needs are.
+
 ## Constraints
 
 - Visibility of emissions data is limited to asset visibility 
-- Emissions Data is updated daily at 04:00 UTC
\ No newline at end of file
+- Emissions Data is updated daily at 04:00 UTC
+- API Rate Limiting: 200 requests per 15 minutes per user
```

Comment:
@alawTrackUnit I have a question about the rate limit. Does the 200 requests per user mean that if a user has visibility of 1000 assets, then they would need to use 5x15min intervals to be able to request emissions data for their entire fleet? Or is it actually a rate limit of 200 requests **per URL** per 15min, which would mean that the user can request the same data for the same asset "only" 200 times per 15min?
If performance of the endpoints allows I would really recommend to allow users to retrieve data for their whole fleet as efficiently as possible.

Reply:
@KateVomBerg We were following the guidelines for this : 
https://trackunit.roadie.so/docs/default/component/architecture-docs/public-api-guidelines/#configuration

We have a max page size of 1000 as well. The default page size is 20.

Reply:
@sieker-trackunit as we discussed the guidelines for rate limits are out-of-date right? Could you guide @alawTrackUnit to set technical rate limits for these endpoints based on the performance under stress test? 🙏🏻 I see no need to enforce any commercial rate limit here, so we can go purely with the needs from a platform stability perspective 💪🏻

Reply:
Hi again @alawTrackUnit, we have now revised the engineering handbook and updated the section for rate limits here: https://github.com/Trackunit/engineering-handbook/blob/master/iris/docs/backend-api-development/public-api-guidelines.md#configuration - apologies for not catching this outdated information earlier 🙏🏻

Reply:
So sorry for the late reply - but luckily Kate updated you. I changed the guideline to reflect what we talked about the other day.
Thanks so much for pointing it out - and please do so again if you have more quesitons 🙏

Reply:
After working with QA we have come up with a rate limit of 1000 calls per 10 mins.

As a side note, do all APIs access the same SQL Server / Instance? Last year we ran into an issue where we had too many API calls to our API and slowed down all of manager, not just our DB instance. If every API has a rate limit they have determined for their own APIs couldn't we end up in a situation where we end up slowing down manager?